migration-guides/0.15-Migration-Guide.md

```diff
@@ -257,3 +265,48 @@ Basic usage:
 parcel build index.html --no-source-maps --no-optimize --no-scope-hoist --dist-dir "dist/" # bundle for the browser
 parcel build index.html --no-source-maps --dist-dir "dist/" # minified bundle for the browser
 ```
+
+## Breaking Changes Made in Core Libraries
+
+### Changes affecting multiple libraries
+
+- Migrated all FFI to ES modules and dropped support for CommonJS modules.
+- Removed all kind-specific Proxy types (e.g. `SProxy`, `Proxy2`, `Proxy3`, `RLProxy`, etc.)
+- Removed `MonadZero` type class and all of its deprecated instances.
+
+### `purescript-prelude` changes
+
+- The data type, `NoConstructors`, often used in `Generic`-related code, was changed to newtype `Void`, enabling one to unwrap the newtype and use `absurd`.
```

Comment:
If this is non-breaking then I'm not sure it needs to be included; if it _is_ breaking then we can keep it.

Reply:
I'm not sure it's breaking, but it's been labeled as such and does actually enable type class instances to exist that weren't possible before. So, I do think it deserves a mention because many do rely upon `Generic`. That being said, I think the main audience would be people using `Generic` to define things like `GenericShow` type classes.